care/emr/reports/context_builder/data_points/fileupload.py

```
@@ -0,0 +1,38 @@
+    QuerysetContextBuilder,
+)
+
+
+class FileUploadReportFilter(filters.FilterSet):
+    file_category = FileCategoryFilter()
+    name = filters.CharFilter(field_name="name", lookup_expr="icontains")
+
+
+class FileUploadContextBuilder(QuerysetContextBuilder):
+    filterset_class = FileUploadReportFilter
+    __filterset_backends__ = [filters.DjangoFilterBackend]
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)